### PR TITLE
LPD-37445 Migrate CannotChangeObjectFromApprovedToDraft

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts [hunk 1]
```diff
@@ -194,6 +194,31 @@ export class PageEditorPage {
 		await this.page.getByRole('option', {name: mode}).click();
 	}
 
+	async changEditableConfiguration({
```
Comment (victorg1991) [inline modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:197]: There is an `e` missing in the `change`  word :P

File: modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts [hunk 2]
```diff
@@ -864,6 +864,12 @@ export class PageEditorPage {
 				.getByRole('menuitem', {name: 'Select Item...'})
 				.click();
 		}
+
+		const iframe = this.page.frameLocator('iframe[title="Select"]');
+
+		await iframe.getByPlaceholder('Search').waitFor();
+
+		return iframe;
```
Comment (victorg1991) [inline modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:872]: This is a bit weird in my opinion, I believe we don't have search bar in all the selectors right? I would move this to the test directly

Comment (veroglez) [reply modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:872]: that's true!

File: modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts (hunk 1 above)
Comment (veroglez) [reply modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts:197]: :sweat_smile:

**Comment by ealonso** [comment]
ci:test:page-management-playwright

**Comment by veroglez** [comment]
ci:stop

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by veroglez** [comment]
ci:stop

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**